scarb/src/bin/scarb/args.rs

```diff
@@ -341,6 +342,8 @@ pub struct FmtArgs {
     /// Specify package(s) to format.
     #[command(flatten)]
     pub packages_filter: PackagesFilter,
+    #[clap(value_name = "TARGET")]
```

Comment:
This should have a doc that will be shown on `scarb fmt --help`. 
Maybe something like: 
```suggestion
    /// The target file for the formatter. If provided, only this file will be formatted.  
    #[clap(value_name = "TARGET")]
```

Reply:
did something similar